assets/css/widescreen.css

```diff
@@ -1,16 +1,638 @@
 body {
   margin: 0;
-  background-color: rgb(255, 255, 255);
 }
 
-html {
-  font-family: serif;
-}
+@media screen and (min-width: 604px) {
```

Comment:
- [ ] The desktop breakpoint for the desktop version is currently too low, it should be, as per requirement,  a value between `992px` and `1100px`, you're currently using `604px`

Reply:
Hi @jenadiusnicholaus just as the previous reviewer requested, use a screen size between 992px and 1100px 
Your breakpoint should look like this if you pick 99px ⬇️ 
@media screen and (min-width: 990px) {
/*
.
.
*/
}
- You don't need to have two screen sizes, all you need is one breakpoint for the desktop version since you followed the mobile-first approach